egs/wsj/s5/utils/validate_disambig_sym_file.pl

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env perl
```

Comment:
we are no longer putting stuff in the top-level utils/ directory-- please locate this in utils/lang.